postgrest/__init__.py

```diff
@@ -29,3 +27,4 @@
 from .deprecated_client import Client, PostgrestClient
 from .deprecated_get_request_builder import GetRequestBuilder
 from .exceptions import APIError
+from .version import __version__
```

Comment:
This line doesn't seem to gel well with build

Reply:
Yeah I should have ran the tests locally first.